evil-collection.el

```diff
@@ -363,6 +363,26 @@ to filter keys on the basis of `evil-collection-key-whitelist' and
                                 err))))))
              (add-hook 'after-load-functions fun t))))))
 
+(cl-defun evil-collection-translate-key-to-state (state
+                                                  new-state
```

Comment:
It seems that `state` and `new-state` are always `nil` and `'emacs` respectively.  Can we remove these parameters then?